### PR TITLE
Add-on EKCO podImageOverrides property

### PR DESCRIPTION
**Pull request by emosbaugh**

### CODE REVIEW

File: src/components/Kurlsh.js [hunk 1]
```diff
@@ -945,6 +945,10 @@ class Kurlsh extends React.Component {
             const currentOption = find(advancedOptions[addOn], (key, value) => value === data.flag);
             const doesCurrentErrExist = installerErrMsg ? installerErrMsg.includes(data.flag) : false;
 
+            if (option.hidden) {
+              return null;
+            }
+
```
Comment (laverya) [inline src/components/Kurlsh.js:951]: should this change be included? seems less related

**Review by laverya — DISMISSED** [review]

File: src/components/Kurlsh.js (hunk 1 above)
Comment (emosbaugh) [reply src/components/Kurlsh.js:951]: It hides the new option from the configuration screen. see https://github.com/replicatedhq/kurl.sh/pull/737/files#diff-edf7ecf74d0199ed4b45aea569c4030ab77a0a29ffb52af50d6c62d15f3fb885R548

**Comment by emosbaugh** [comment]
[sc-37396]

**Review by jonquil2002 — APPROVED** [review]
@emosbaugh - Approved with my commits, unless I rendered something as technically incorrect.

Please merge when you are ready. Thanks.

**Pull request merged**